app/services/client_logins_service.rb

```diff
@@ -61,11 +61,11 @@ def create_text_message_login(sms_phone_number:, visitor_id:, locale:)
 
     def clients_for_token(raw_token)
       # these might have multiple email addresses
-      to_addresses = EmailAccessToken.by_raw_token(raw_token).pluck(:email_address)
+      to_addresses = EmailAccessToken.lookup(raw_token).pluck(:email_address)
```

Comment:
Now I'm kind of thinking would be nice to have AccessToken be an STI table so that we could do one query to look for matches on either phone numbers or emails instead of querying across multiple tables.

Reply:
That's interesting! Not a bad idea.

I also looked at [Delegated Types](https://guides.rubyonrails.org/6_1_release_notes.html#delegated-types) recently and thought they were cool.

Reply:
If they were STI, then I think we couldn't make `email_address` `NOT NULL` in the DB, and I love my database constraints.